cookbooks/global_melt.prm

```diff
@@ -304,12 +304,10 @@ subsection Mesh refinement
 
   # minimum of 4 global refinements
   subsection Minimum refinement function
-    set Coordinate system   = depth
     set Function expression = 4
-    set Variable names      = depth,phi
```

Comment:
Can you say why these changes are correct?

Reply:
Ah, thanks for the explanation across the table: The function expression is a constant, there is no point in naming coordinates explicitly.